source/Visibility/ArcMapAddinVisibility/ViewModels/RLOSViewModel.cs

```diff
@@ -22,10 +28,9 @@
 using ESRI.ArcGIS.Geodatabase;
 using ESRI.ArcGIS.Geometry;
 using ESRI.ArcGIS.Geoprocessing;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Windows;
+
+// Solution
+using VisibilityLibrary.Helpers;
```

Comment:
Look at DistanceDirection solution.  Most using statements are commented for the sections that they represent.  Keeps things in sections as opposed to them all being dispersed with each other.